centos/base/Dockerfile

```diff
@@ -0,0 +1,216 @@
+FROM centos:8 AS base
+
+# Installs all necessary development packages for a full python compilation.
+RUN dnf update -y \
+    && dnf groupinstall -y "development tools" \
+    && dnf install epel-release -y \
+    && dnf -y install dnf-plugins-core -y \
+    && dnf config-manager --set-enabled powertools -y \
+    && dnf install -y \
+        wget \
+        gcc \
+        openssl-devel \
+        bzip2-devel \
+        libffi \
+        libffi-devel \
+        epel-release \
+        crudini \
+        wget \
+        zlib-devel \
+        ncurses-devel \
+        sqlite-devel \
+        readline-devel \
+        tk-devel \
+        gdbm-devel \
+        libpcap-devel \
+        xz-devel \
+        expat-devel \
+        diffutils
+
+## Installs Python3.7 from source - this is for running Packs with Python other than 3.6
+RUN wget https://www.python.org/ftp/python/3.7.6/Python-3.7.6.tar.xz \
+    && tar -xf ./Python-3.7.6.tar.xz \
+    && cd ./Python-3.7.6 \
+    && ./configure \
+        --enable-optimizations \
+        --with-ensurepip=install \
+        # --enable-loadable-sqlite-extensions \
+        # --enable-shared \
+        # --with-system-expat \
+        # --with-system-ffi \
+    && make -j "$(nproc)" \
+    && cd ..
```

Comment:
This could really be any version of python. We have been running a separate python since early stackstorm at py2.x; so we built our packs with the 'major' python version at the time, py3.7.

though stackstorm itself runs with the py3.6 that the rpm installs with it. 

yes, it would be great if there were updated EL8 packages for python (3.7, 3.8, etc...)